package/yast2-migration-sle.spec

```diff
@@ -0,0 +1,70 @@
+#
+# spec file for package yast2-migration-sle
+#
+# Copyright (c) 2022 SUSE LLC, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Name:           yast2-migration-sle
+Version:        4.4.0
+Release:        0
+Summary:        YaST2 - Online migration
+Group:          System/YaST
+License:        GPL-2.0
+Url:            https://github.com/yast/yast-migration-sle
+
+Source0:        %{name}-%{version}.tar.bz2
+
+BuildRequires:  yast2-buildtools
+BuildRequires:  yast2
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:rspec)
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:yast-rake)
+BuildRequires:  yast2-ruby-bindings
+BuildRequires:  update-desktop-files
+
+Requires:       yast2
+Requires:       yast2-ruby-bindings
+Requires:       yast2-registration
+Requires:       yast2-migration
+
+Supplements:    yast2-registration
+
+BuildArch: noarch
+
+%description
+This package contains the YaST2 component for migrating an openSUSE
+system to the SUSE Linux Enterprise (SLE) system.
```

Comment:
More specifically to SLES, right?

Reply:
Um, currently yes, but in theory you could migrate to SLE4SAP, SLES-HPC,... it just depends what SCC returns.